src/util/bip143.rs

```diff
@@ -107,19 +107,19 @@ impl SighashComponents {
 }
 
 /// A replacement for SigHashComponents which supports all sighash modes
-#[deprecated(since = "0.27.0", note = "please use [sighash::SigHashCache] instead")]
-pub struct SigHashCache<R: Deref<Target = Transaction>> {
-    cache: sighash::SigHashCache<R>,
+#[deprecated(since = "0.27.0", note = "please use [sighash::SighashCache] instead")]
+pub struct SighashCache<R: Deref<Target = Transaction>> {
```

Comment:
I think we should revert this change. It does not make sense to rename something that is deprecated.

Reply:
ooo yes, my bad. Thanks!

Reply:
Removed patch: `eaf2040 Rename SigHashType -> SighashType`, win for separate patches!

Reply:
Fixed `SigHashCache` change to only do the non-deprecated type (the one is `sighash` module).